stdlib/public/SwiftShims/SwiftStdint.h

```diff
@@ -24,7 +24,7 @@
 
 // Clang has been defining __INTxx_TYPE__ macros for a long time.
 // __UINTxx_TYPE__ are defined only since Clang 3.5.
-#if !defined(__APPLE__) && !defined(__linux__) && !defined(__OpenBSD__)
+#if !defined(__APPLE__) && !defined(__linux__) && !defined(__OpenBSD__) && !defined(FreeBSD)
```

Comment:
You probably want `defined(__FreeBSD__)` here.

Reply:
Good catch, thanks!